stellargraph/core/graph.py

```diff
@@ -591,10 +603,10 @@ def type_for_node(self, node):
     @property
     def node_types(self):
```

Comment:
The majority of this class does not have type annotation. Why is it important to add it for this function?

Reply:
Seems that if we are going to make a point (in the docs) that the return type is a set, it might be useful to start `typing` that. Not important, just niggled at me for some reason.

Reply:
I think we need to add it to the base `StellarGraph` class if we want that.  For some reason this method is not there.